pkg/shell/state.tsx

```diff
@@ -646,6 +647,23 @@ export class ShellState extends EventEmitter<ShellStateEvents> {
         this.current_manifest_item = item;
         this.current_manifest = compiled.find_path_manifest(location.path);
 
+        this.current_fullscreen = false;
+        if (this.current_manifest.fullscreen) {
+            let fs = this.current_manifest.fullscreen;
+            const path_prefix = location.path.split("/", 1)[0];
+            if (Array.isArray(fs))
+                fs = { "": fs };
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21670-ca14cdd0-20250716-121825-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)